.github/workflows/main.yml

```
@@ -0,0 +1,61 @@
+name: transfer-repo-CI
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build/Test
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        go-version: [1.16.2]
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: checkout and update submodules
+        run: git submodule update --init --recursive --remote
```

I don't think we want `--remote`. This could potentially cause CI to run using an upstream version of the submodule relative to where it was specified in the superproject. I suppose this could become a non-issue if we had a convention around specifying submodule HEADs (e.g. using a hash instead of a branch name).

From `git help submodule`:
```
--remote
   This option is only valid for the update command. Instead of using the superproject’s
   recorded SHA-1 to update the submodule, use the status of the submodule’s
   remote-tracking branch. The remote used is branch’s remote (branch.<name>.remote),
   defaulting to origin. The remote branch used defaults to the remote HEAD, but the
   branch name may be overridden by setting the submodule.<name>.branch option in either
   .gitmodules or .git/config (with .git/config taking precedence).

```

```suggestion
        run: git submodule update --init --recursive
```